process_manual/project_meetings.md

```diff
@@ -133,3 +147,20 @@ The following, generic, options were given:
 This meant that every developer had to give their opinion on every problem or solution, and it revealed some controversial topics.
 
 We, the process group, spent the rest of the day discussing changes to the process, and presented it the next day to the sprint intro.
+
+## Sprint intro
```

Comment:
I fell like this is kind of unnecessary and just confusing, because we no longer use it, either move it to another file or something